docs: move 'What can I make with SvelteKit' back to FAQ (#10547)
sveltejs/kit: documentation/docs/60-appendix/05-integrations.md

## Integration FAQs

> The SvelteKit FAQ has a [section on integrations](./faq#how-do-i-use-x-with-sveltekit), which may be helpful if you still have questions.

Reviewer: ```suggestion
The SvelteKit FAQ has a [how to do X with SvelteKit](./faq#how-do-i-use-x-with-sveltekit), which may be helpful if you still have questions.
```

This link might make a bit more sense for this page

Author: Ready!
